RFC/src/RFC-0301_NamespaceRegistration.md

```diff
@@ -171,26 +191,22 @@ as follows:
 - The `Hash256` algorithm is not prescribed, but MUST be consistently used and MUST produce a 64 character (256 bit) 
   hexadecimal output.
 - Stage 1 - MUST select the input string to use (either `"No FQDN"` or `PubKey || <FQDN>`).
```

Comment:
I propose an alternative algorithm to address 2 issues:
1. This forces us to code a bunch of Bitcoin mechanics that will not be used anywhere else in the system; Base58 is quite useful, but I don't see another use for ripe160
2. We're trying to get a 16 char RAID_ID

Alternative algo:
1. Select FQDN||Default as before as a little endian byte string 
1. `H(FQDN)` with Blake2b using 10 byte digest
1. Add `0x62` prefix
1. Base58 encode to 15 char string starting with R

https://gist.github.com/CjS77/9dc02ef008522ce0a7d5357551010726

Reply:
Very promising @CjS77! 
Looking into this, trying to run your Python script, but having issues installing the bitcoin environment with `pip install bitcoinlib`.

Reply:
RAID_ID generation updated as suggested